src/sctp_transport/mod.rs

```diff
@@ -212,12 +212,22 @@ impl RTCSctpTransport {
     }
 
     async fn accept_data_channels(param: AcceptDataChannelParams) {
+        let dcs = param.data_channels.lock().await;
+        let mut existing_data_channels = Vec::new();
+        for dc in dcs.iter() {
+            if let Some(dc) = dc.data_channel.lock().await.clone() {
+                let dc2 = dc.as_ref().clone();
+                existing_data_channels.push(dc2);
+            }
+        }
+
         loop {
             let dc = tokio::select! {
                 _ = param.notify_rx.notified() => break,
                 result = DataChannel::accept(
                     &param.sctp_association,
                     data::data_channel::Config::default(),
+                    existing_data_channels.to_owned(),
```

Comment:
why do we need `to_owned`?

Reply:
(sorry for slow response!) 

without `to_owned()` we get an error of the `existing_data_channels` value moving in a previous loop iteration. I'll readily admit that I am no Rust expert though, so if there's a better solution to this then I'm all ears :)